tools/api-markdown-documenter/src/documentation-domain-to-html/ToHtml.ts

```diff
@@ -47,32 +47,80 @@ export function documentToHtml(document: DocumentNode, config: TransformationCon
 }
 
 /**
- * Renders the provided {@link DocumentationNode} per the configured
- * {@link HtmlRenderContext.customRenderers | renderers}.
+ * Generates an HTML AST from the provided {@link DocumentationNode}.
  *
- * @alpha
+ * @param node - The documentation node to transform.
+ * @param config - The HTML transformation configuration. Unspecified options will be filled with defaults.
+ *
+ * @public
+ */
+export function documentationNodeToHtml(
+	node: DocumentationNode,
+	config: TransformationConfig,
+): HastNodes;
+/**
+ * Generates an HTML AST from the provided {@link DocumentationNode}.
+ *
+ * @param node - The documentation node to transform.
+ * @param context - The HTML transformation context.
+ *
+ * @public
  */
 export function documentationNodeToHtml(
 	node: DocumentationNode,
 	context: TransformationContext,
+): HastNodes;
+/**
+ * Generates an HTML AST from the provided {@link DocumentationNode}.
+ */
+export function documentationNodeToHtml(
```

Comment:
This is cool... so we have the "superset signature" because we need it but only the ones marked `@public` are exported, correct?

Reply:
It actually isn't related to the tagging, though it may be possible to tag different overloads differently, I haven't tried that. I bet that works, and could be a cool way to expose variants of functions... I should try that...

The base implementation of a series of overloads doesn't get a type signature generated by TypeScript, so it wouldn't appear in the rollup regardless of how it is tagged. The only reason it has any documentation at all is because our linter required it, and it didn't seem worth the disable pragma to avoid 😆

Reply:
> It actually isn't related to the tagging, though it may be possible to tag different overloads differently, I haven't tried that. I bet that works, and could be a cool way to expose variants of functions... I should try that...

Dang, no it doesn't work. API-Extractor requires all symbols of the same name be at the same release level. That's a bummer. That would have been really cool.